org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/XMLLanguageServer.java

```diff
@@ -64,9 +72,29 @@ public XMLLanguageServer() {
 		capabilities.setHoverProvider(true);
 		// capabilities.setExperimental("foldingRangeProvider: true");
 		InitializeResult result = new InitializeResult(capabilities);
+		initializeLoggers(languageClient, params.getInitializationOptions().toString());
 		return CompletableFuture.completedFuture(result);
 	}
 
+	public void initializeLoggers(LanguageClient newLanguageClient, String path) {
+		path = path.replaceAll("\"", "");
+		Logger logger = Logger.getLogger("");
+		logger.setLevel(Level.INFO);
+		logger.setUseParentHandlers(false);// Stops output to console
+		String newPath = Paths.get(path).normalize().getParent().toString();
+		new File(newPath).mkdirs();
+		try {	
+			FileHandler fh = new FileHandler(path, false);

```

Comment:
new FileHandler(path, true);

Reply:
How do log files usually work in terms of when they get should cleaned out?